spec/acceptance/ferm_spec.rb

```diff
@@ -43,12 +53,36 @@ class { 'ferm':
       },
     },
     ip_versions      => ['ip'], #only ipv4 available with CI
-  }
```

Comment:
why did you delete this line?

Reply:
to factorize the manifest and add at this end some parameters or not